Spielregeln.md

```diff
@@ -0,0 +1,88 @@
+
+(3) Das Ziel muss in Größe und Form einer 1,0-Liter-Plastikflasche ähneln.
+Es muss in einer Art beschwert sein (z. B. durch das Befüllen mit Wasser), dass es nicht durch leichte Krafteinwirkung, wie z. B. Windstöße umfällt.
+Allerdings ist sicherzustellen, dass es möglich ist, das Ziel mit dem Wurfgerät zu Fall zu bringen.
+
+(4) Getrunken wird aus Glasflaschen der 0,5-Liter-Klasse, deren Öffnung DIN EN ISO 12822 oder DIN EN ISO 12821 (handelsübliche Bierflasche mit Kronkorkenverschluss) entspricht.
+Die Getränke müssen Kohlensäure enthalten.
+Es ist explizit erlaubt, Getränke zu konsumieren die keinen Alkohol enthalten bzw. nicht als Bier klassifiziert werden.
+
+## § 4 Spielfeld
+
+(1) In der Mitte des Spielfeldes wird das Ziel aufgestellt.
+
+(2) Von dort ausgehend sind in entgegengesetzte Richtung 6 m (~7 Schritte) abzumessen an denen sich die jeweiligen Grundlinien (auch "Wurflinien") der zwei Teams befinden.
+
+(3) Die Grundlinien der Teams verlaufen parallel.
+
+## § 5 Ablauf
+
+(1) Es gibt immer genau ein Team, welches versucht das Ziel mithilfe des Wurfgeräts zu Fall zu bringen ("werfendes Team") und ein Team, welches darauf reagiert ("aufstellendes Team").
+Nach jedem Wurf wird gewechselt.
+
+(2) Vor Beginn entscheidet ein Münzwurf oder eine Partie Schere-Stein-Papier zwischen zwei Mitgliedern der spielenden Teams über den ersten Wurf.
+Das gewinnende Team entscheidet, welches Team zuerst wirft.
+
+(3) Vor jedem Wurf, muss sichergestellt werden, dass das gegnerische Team bereit ist.
+Bei Missachtung dieser Regel wird der Wurf annulliert und das Wurfrecht geht an das gegnerische Team über.
+
+(4) Geworfen wird stehend und von oben, d. h. der Ball muss die Wurfhand ungefähr auf Schulterhöhe verlassen.
+Die Grundlinie darf dabei nicht übertreten werden.
+Bei Missachtung dieser Regel wird der Wurf annulliert und das Wurfrecht geht an das gegnerische Team über.
+
+(5) Der Rest des werfenden Teams ist in der Wahl der Körperhaltung frei, darf die Getränkebehältnisse allerdings nicht vor Fall des Zieles berühren.
+
+(6) Es ist erlaubt, seine Getränke vor Umfallen zu schützen.
+
+(7) Sollte das Ziel umfallen, bevor das Wurfgerät die Grundlinie des aufstellenden Teams passiert hat, darf das werfende Team mit dem Trinken beginnen.  
+Getrunken wird so lange, bis das Ziel erfolgreich aufgestellt wurde und sich alle Mitglieder des aufstellenden Teams mitsamt des Wurfgeräts hinter ihrer Grundlinie befinden.
+Der Abschluss dieses Vorgangs ist mit einem deutlich hörbaren "Stop"-Ruf anzuzeigen.  
+Das werfende Team muss unverzüglich alle Trinkaktionen einstellen und die Flaschen zurück auf den Boden führen.
+
+## § 6 Strafen
+
+(1) Im Fall, dass ein Getränk überschäumt, erhält das gegnerische Team die Möglichkeit einen extra Schluck aus dem Getränkebehältnis zu nehmen ("Stafschluck").
+Es darf nur so viel getrunken werden, dass exakt eine Schluckbewegung ausgeführt wird.
```

Comment:
Es ist schwierig zu kontrollieren, ob der Gegenüber genau eine Schluckbewegung macht. Beziehungsweise ist es als Trinkender schwierig, die Menge einer Schluckbewegung abzuschätzen.
Wie wärs mit `Es darf nur so viel getrunken werden, dass der Mundraum einmalig befüllt wird.`

Reply:
Man kann in diesem Spiel so vieles nicht kontrollieren. Mundraum füllen halte ich für etwas krass. Manche Leute haben echt große Münder (:smile:) und in jedem Fall kann man dadurch deutlich mehr trinken, als wenn man nur ein Mal schluckt. 
Ich weiß nicht, ob die Player-Base reblliert, wenn wir den Strafschluck so hart buffen. Könnte die Meta ganz schön aus dem Gleichgewicht bringen :smile:

Reply:
ganz ehrlich, wenn man dann mit einem Strafschluck weniger trinkt als man könnte, hat man halt Pech. Da wird man aber an sich über die Zeit auch immer besser mit. Also ich finde wir sollten das so lassen. Wahrscheinlich sollten wir nur irgendwo festhalten, dass der Schiri wiederholte und böswillige Verstöße gegen die Regel ahnden darf und soll.